Formula/sourcedocs.rb

```diff
@@ -0,0 +1,21 @@
+class Sourcedocs < Formula
+  desc "Generate Markdown files from inline source code documentation"
+  homepage "https://github.com/eneko/SourceDocs"
+  url "https://github.com/eneko/SourceDocs/archive/0.5.1.tar.gz"
+  sha256 "3c2e2de695d49dbdd5acb49f8876042bdc97e8d6b95584d3ef6b592b8f10affc"
+  head "https://github.com/eneko/SourceDocs.git"
+
+  depends_on :xcode
+
+  def install
+    build_path = "#{buildpath}/.build/release/sourcedocs"
+    ohai "Building SourceDocs"
+    system("swift build --disable-sandbox -c release -Xswiftc -static-stdlib")
+    ohai "Installing SourceDocs..."
+    bin.install build_path
+  end
+
+  test do
+    system "#{bin}/sourcedocs", "version"
```

Comment:
Please write a test that exercises a deeper level of the functionality provided by the formula.

Reply:
This might take a while, I'll will work on some tests to validate the generator functionality. Meanwhile, I've updated this test to validate the version number.